
---
geometry_msgs/PoseArray truck_pose
